src/rendition-mixin.js

```diff
@@ -11,21 +12,22 @@
  * or if undefined returns the current enabled-state for the playlist
  * @return {Boolean} The current enabled-state of the playlist
  */
-let enableFunction = (playlist, changePlaylistFn, enable) => {
-  let currentlyEnabled = typeof playlist.excludeUntil === 'undefined' ||
-                         playlist.excludeUntil <= Date.now();
+const enableFunction = (loader, playlistUri, changePlaylistFn, enable) => {
+  const playlist = loader.master.playlists[playlistUri];
```

Comment:
Yes, the original function used just the playlist but this PR needed to change that since the call to `bind` below would reference an outdated playlist object since `PlaylistLoader` overwrites the reference when it gets a new playlist. This ensures we are acting on the correct object.